join.go

```diff
@@ -12,39 +12,26 @@ package securejoin
 
 import (
 	"bytes"
+	"errors"
 	"os"
 	"path/filepath"
 	"strings"
 	"syscall"
-
-	"github.com/pkg/errors"
 )
 
 // ErrSymlinkLoop is returned by SecureJoinVFS when too many symlinks have been
 // evaluated in attempting to securely join the two given paths.
-var ErrSymlinkLoop = errors.Wrap(syscall.ELOOP, "secure join")
+//
+// Deprecated: use errors.Is(err, syscall.ELOOP) instead.
+var ErrSymlinkLoop = syscall.ELOOP
```

Comment:
I was not able to find any external user of this, so maybe it's better to just remove this?

Reply:
Removed

Reply:
Yeah I don't think anybody used it, but I added it so we could detect it in principle. `errors.Is(err, unix.ELOOP)` works now and is more idiomatic anyway.